### PR TITLE
MH-60R migrated to module

### PR DESCRIPTION
**Pull request by jdahlblom**
Not tested. Straightforward migration though.

### CODE REVIEW

File: Scripts/DCS-BIOS/BIOS.lua [hunk 1]
```diff
@@ -120,6 +120,9 @@ dofile(lfs.writedir()..[[Scripts/DCS-BIOS/lib/M-2000C.lua]]) -- ID = 27, ProperN
 -- dofile(lfs.writedir()..[[Scripts/DCS-BIOS/lib/MB-339.lua]]) -- ID = 28, ProperName = MB-339A/PAN
 local MB_339 = require "MB-339"
 BIOS.protocol.writeNewModule(MB_339)
+-- dofile(lfs.writedir()..[[Scripts/DCS-BIOS/lib/archive/old_format_planes/MH-60R.lua]]) -- ID = 42, ProperName = Mi-24P HIND
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/BIOS.lua:123]: Id and Proper Name are copied from hind

File: Scripts/DCS-BIOS/doc/json/AircraftAliases.json [hunk 2]
```diff
@@ -64,6 +64,7 @@
                "MB-339A": [ "CommonData", "MB-339" ],
             "MB-339APAN": [ "CommonData", "MB-339" ],
              "MB-339PAN": [ "CommonData", "FC3" ],
+                "MH-60R": [ "MH-60R" ],
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/doc/json/AircraftAliases.json:67]: Should also have commondata. I'm guessing this needs to be added to AircraftList.lua as well.

**Review by charliefoxtwo — CHANGES_REQUESTED** [review]
Just needs to be added to AircraftList.lua (and I think there might be a text file somewhere that stores the ids as well? I don't remember what it's called though)

**Comment by charliefoxtwo** [comment]
Here we go - it should be added here as well https://github.com/DCS-Skunkworks/dcs-bios/blob/master/Scripts/DCS-BIOS/doc/addresses.txt

File: Scripts/DCS-BIOS/BIOS.lua (hunk 1 above)
Comment (jdahlblom) [reply Scripts/DCS-BIOS/BIOS.lua]: fixed

File: Scripts/DCS-BIOS/doc/addresses.txt [hunk 3]
```diff
@@ -47,6 +47,7 @@ FC3				0x6000	#4
 AH-6J				0x6200	#7		(Mod)
 Alphajet			0x6400	#9		(Mod)
 F-22A				0x6600	#40		(Mod)
+MH-60R				0xA400	#47
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/doc/addresses.txt:50]: These should be in order by address I believe

Comment (charliefoxtwo) [reply Scripts/DCS-BIOS/doc/addresses.txt:50]: There's already a line for 0xa400 so just add it there

Comment (jdahlblom) [reply Scripts/DCS-BIOS/doc/addresses.txt:50]: yes and those #id should be removed, they are for DCSFP but it has it's own file now.

Comment (charliefoxtwo) [reply Scripts/DCS-BIOS/doc/addresses.txt:50]: Just check with @WarLord211 on this one, he typically maintains this file. I don't use it personally.

Comment (charliefoxtwo) [reply Scripts/DCS-BIOS/doc/addresses.txt]: I think the ID in the bios.lua comment for the line is supposed to match the # here as well (again, not sure why)

Comment (jdahlblom) [reply Scripts/DCS-BIOS/doc/addresses.txt]: yes but they are not necessary anymore, DCSFP don't read any module info from DCS-BIOS anymore. Manually entered into a file. User can do that if a module is missing until it is added in the release.

File: Scripts/DCS-BIOS/doc/json/AircraftAliases.json [hunk 4]
```diff
@@ -119,6 +120,7 @@
                   "T-45": [ "CommonData", "VNAO_T-45" ],
                 "TF-51D": [ "CommonData", "NS430", "P-51D" ],
                  "UH-1H": [ "CommonData", "NS430", "UH-1H" ],
+                "UH-60L": [ "MH-60R" ],
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/doc/json/AircraftAliases.json:123]: Shouldn't this also show Commondata? You likely need to add this to AircraftList.lua as well

**Review by charliefoxtwo — APPROVED** [review]

**Pull request merged**